core/cap-0046-06.md

```diff
@@ -136,8 +136,8 @@ pub enum SignaturePayload {
 
 // Initializes a token contract that does not wrap an asset on the classic side.
 // Should be used on a contract created with create_token_from_contract.
-// Sets admin, decimals, name, and symbol
-fn init(admin: Identifier, metadata: TokenMetadata) -> Result<(), Error>;
+// Sets admin, decimals, name, symbol, and the clawback flag.
+fn init(admin: Identifier, metadata: TokenMetadata, can_clawback: bool) -> Result<(), Error>;
```

Comment:
nit: I think `can_clawback` should probably just go into `TokenMetadata` (on a side note it should probably described in the CAP as it's a part of the public interface)

Reply:
I kept it out because it's mutable (can be flipped off) unlike the fields in TokenMetadata. I don't feel strongly about this though. We can change this later if we want to.